doc/BuildingWithCMake.md

```diff
@@ -54,6 +54,104 @@ CMake can generate build files for a number of backends. You can select the gene
 available generators with `--help`. We recommend using with the `Ninja` generator, when available. More information is
 available [here](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html).
 
+## Configuring the build on Windows
+
+There are three widely used C/C++ toolchains for Windows: *MSYS2/MinGW*, *Microsoft Visual Studio* and *LLVM/Clang*.
+Let's consider how to use each toolchain to configure and build `OMR` with `CMake`.
+
+### Common prerequisite
+
+*Perl* must be installed otherwise `CMake` won't be available to configure the project:
+
+```bash
```

Comment:
Perl is needed on every platform, can you move it to the common prerequisites even earlier in this document? As well, I don't think it's particularly useful to show the "Perl not found" error message.

Reply:
Maybe I don't understand your comment well: on Linux platforms, Perl usually is already installed, I'm putting only the requirements for Windows. As I see there is a bit of confusion because I use a wrong header level for my Windows-specific instruction. Fixed.

Reply:
I really did mean for you to move perl to the general dependencies list at the top of the document. Maybe it's pedantic but, perl not being there in the first place was an oversight (even if the dependency is normally preinstalled on linux).

If you want to recommend strawberry perl here in the windows section, that's fine.

The "Could not find Perl" cmake output does not need to be in this document.

Reply:
Ok, I've removed this code section and left strawberry perl in the windows section.